sys/include/net/gnrc/ipv6/ext.h

```diff
@@ -79,6 +53,66 @@ void gnrc_ipv6_ext_demux(gnrc_netif_t *netif,
 gnrc_pktsnip_t *gnrc_ipv6_ext_build(gnrc_pktsnip_t *ipv6, gnrc_pktsnip_t *next,
                                     uint8_t nh, size_t size);
 
+#if     defined(MODULE_GNRC_IPV6_EXT) || defined(DOXYGEN)
+/**
+ * @brief   Processes a packet's payload as hop-by-hop option if @p nh is
+ *          pointing to a value equal to @ref PROTNUM_IPV6_EXT_HOPOPT.
+ *
+ * @note    Should be called by @ref net_gnrc_ipv6 before any further processing
+ *          (even forwarding-related) is done to @p pkt.
+ *
+ * @param[in] pkt       An IPv6 packet in receive order. It assumed that
+ *                      gnrc_pktsnip_t::data points to a hop-by-hop option when
+ *                      @p nh points to value equal to
+ *                      @ref PROTNUM_IPV6_EXT_HOPOPT.
+ * @param[in,out] nh    **In:** The @ref net_protnum of gnrc_pktsnip_t::data of
+ *                      @p pkt. <br />
+ *                      **Out:** If @p nh was pointing to a value equal
+ *                      to @ref PROTNUM_IPV6_EXT_HOPOPT on in and the return
+ *                      value is not NULL it points to the next header field
+ *                      of the processed hop-by-hop option header, if the return
+ *                      value is NULL the value undefined.
```

Comment:
missing "is"

```suggestion
 *                      value is NULL the value is undefined.
```

Reply:
Also, "if the value is [...] the value is undefined" is a tad confusing, I'd specify *which* value remains undefined by saying something like `value is NULL the @p nh value is undefined`